src/Documentation/sidebar.json

```diff
@@ -24,14 +24,42 @@
       {
         "label": "Get Older Files",
         "slug": "older-versions"
+      }
+    ]
+  },
+  {
+    "slug": "tutorials",
+    "source": "tutorials/index.md",
```

Comment:
Related: Do we need to move [`tutorial`](https://github.com/iterative/dataset-registry/tree/master/tutorial) to "`tutorials`" (plural) in the `dataset-registry` repo? And rename/complete the internal directories `nlp`, `ver`, etc. Please lmk what it should look like to match this PR.

Reply:
> Related: Do we need to move tutorial to "tutorials" (plural) in the dataset-registry repo? And rename/complete the internal directories nlp, ver, etc. Please lmk what it should look like to match this PR.

I don't think it needs to be changed. At least that change does not seem to affect this PR.

Reply:
Agreed. Moved to https://github.com/iterative/dataset-registry/issues/3